lang/fr.js

```diff
@@ -1,37 +1,39 @@
 /* eslint quotes: 0 */
 
 export default {
-	"components.alert.close": "Fermer l'alerte",
+	"components.alert.close": "Fermer l\'alerte",
 	"components.breadcrumbs.breadcrumb": "Chemin de navigation",
 	"components.calendar.notSelected": "Non sélectionné(e)",
 	"components.calendar.selected": "Sélectionné(e).",
 	"components.calendar.show": "Afficher {month}",
 	"components.dialog.close": "Fermer cette boîte de dialogue",
-	"components.form-element.defaultError": "{label} n'est pas valide.",
+	"components.form-element.defaultError": "{label} n’est pas valide.",
 	"components.form-element.defaultFieldLabel": "Champ",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.url.typeMismatch": "",
-	"components.form-element.valueMissing": "{label} est requis(e).",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-element.input.email.typeMismatch": "{label} is not a valid email",
+	"components.form-element.input.number.rangeOverflow": "Number must be lower than {max}",
+	"components.form-element.input.number.rangeUnderflow": "Number must be higher than {min}",
+	"components.form-element.input.url.typeMismatch": "{label} is not a valid URL",
+	"components.form-element.valueMissing": "{label} est requis.",
+	"components.form-error-summary.errorSummary": "There {count, plural, one { was 1 error } other { were { count } errors }} found in the information you submitted",
 	"components.input-date-range.endDate": "Date de fin",
-	"components.input-date-range.errorBadInput": "",
+	"components.input-date-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-date-range.startDate": "Date de début",
 	"components.input-date-time-range.endDate": "Date de fin",
 	"components.input-date-time-range.startDate": "Date de début",
 	"components.input-date-time.date": "Date",
 	"components.input-date-time.time": "Heure",
 	"components.input-date.clear": "Effacer",
-	"components.input-date.errorMaxDateOnly": "",
-	"components.input-date.errorMinDateOnly": "",
-	"components.input-date.errorOutsideRange": "Doit être entre le {minDate} et le {maxDate}",
+	"components.input-date.errorMaxDateOnly": "La date doit être antérieure à {maxDate}",
+	"components.input-date.errorMinDateOnly": "La date doit être postérieure à {minDate}",
+	"components.input-date.errorOutsideRange": "La date doit être comprise entre {minDate} et {maxDate}",
```

Comment:
went from "Must be between {minDate} and {maxDate}" to "
The date must be between {minDate} and {maxDate}". Seems fine and consistent with the other translations.